cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -124,11 +124,14 @@ def children_records(self, key, value):
 def access_urls(self, key, value):
     """Translates access urls field."""
     _access_urls = self.get("access_urls", [])
-
+    access_type_mapped = []
+    access_type_array = list(map(int, str(clean_val("x", value, str))))
```

Comment:
naming police 🚓 
access_type_array -> access_type_list

also, for this line:
```
str(clean_val("x", value, str)
```
clean_val function already returns str type (or None if empty) there is no need to cast it to string twice